Frontend/src/Components/PPC_MPPT/PPC_MPPT.js

```diff
@@ -34,43 +37,135 @@ export default function PPC_MPPT(props) {
                     <Spacer/>
                     <Flex w='45%' direction='column'>
                         <MPPT_CELL
-                            label='Charge'
+                            label='Tracking'
                             isCurrMode={props.data?.mppt_mode[0]}
                         />
                         <Spacer/>
                         <MPPT_CELL
-                            label='MPPT'
+                            label='Current Control'
                             isCurrMode={!props.data?.mppt_mode[0]}
                         />
                     </Flex>
                 </Flex>
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 1 Temp'
-                    data={props.data?.string1_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string1_temp.MIN}
-                    max={CONSTANTS.string1_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string2_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string2_temp.MIN}
-                    max={CONSTANTS.string2_temp.MAX}
-                />
-                <RangeCell
-                    fontSize='1vw'
-                    label='Cell String 2 Temp'
-                    data={props.data?.string3_temp[0] ?? -1.0}
-                    digits={3}
-                    unit="&#8451;"
-                    min={CONSTANTS.string3_temp.MIN}
-                    max={CONSTANTS.string3_temp.MAX}
-                />
+                <Grid
+                    py='1rem'
+                    templateColumns="repeat(4, 1fr)"
+                    templateRows="repeat(4, 1fr)"
+                    gap={0.5}
+                >
+                    <Spacer></Spacer>
+                    <GridItem
+                        bg='gray'
+                        color='black'
+                        fontWeight='bold'
+                        height='20px'
+                        fontSize={10}
+                        alignItems='center'
+                        display='flex'>Temp(ºC)</GridItem>
+                    <GridItem
+                        bg='gray'
+                        color='black'
+                        fontWeight='bold'
+                        height='20px'
+                        fontSize={10}
+                        alignItems='center'
+                        display='flex'>Voltage(V)</GridItem>
+                    <GridItem
+                        bg='gray'
+                        color='black'
+                        fontWeight='bold'
+                        height='20px'
+                        fontSize={10}
+                        alignItems='center'
+                        display='flex'>Current(A)</GridItem>
```

Comment:
```suggestion
                    <GridItem
                        bg='gray' THIS (let's use headerBg, which already exists in colors.js, instead of gray)
                        color='black' THIS (we should get rid of this)
                        fontWeight='bold'
                        height='20px' THIS (mostly for consistency, let's translate this to vh)
                        fontSize={10} THIS (since we use vw for fontSize in RangeCell, let's do the same in GridItems)
                        alignItems='center'
                        display='flex'>Current(A)</GridItem>
```
A few things:
1. Using static colors doesn't generally work well with both dark and light color modes, so we should avoid doing it wherever possible. I mentioned something similar, as well as gave some points on fixing it, in a [comment in Sarthak's convertData PR](https://github.com/badgerloop-software/chase-car-dashboard/pull/49#discussion_r1181289415).
2. For the `height` and `fontSize`, we should avoid static sizes like `px` (and bare units which are px), unless there's 0 chance of overflow. I know it's not a proper solution, but for consistency's sake, let's use `vh` and `vw` for `height` and `fontSize`, respectively. Same goes for other static sizes used in this component.